framework/src/governance/lib/data-catalog-database.ts

```diff
@@ -23,6 +26,10 @@ import { Context, TrackedConstruct, TrackedConstructProps, Utils } from '../../u
  * });
  */
 export class DataCatalogDatabase extends TrackedConstruct {
+  /**
+   * Default permission model for the DataCatalogDatabase
+   */
+  private static readonly DEFAULT_PERMISSION_MODEL = PermissionModel.IAM;
```

Comment:
Can you throw an error if the permission model is IAM but the user pass the roles for lakeformation.